install/kubernetes/cilium/values.yaml.tmpl

```
@@ -891,6 +891,9 @@ hubble:
       # -- Metrics relabeling configs for the ServiceMonitor hubble
       metricRelabelings: ~
     dashboards:
+      # -- Grafana dashboards for hubbel
```

```suggestion
      # -- Grafana dashboards for hubble
```

Interestingly enough it's correctly spelled in `values.yaml`. Maybe run `make -C install/kubernetes` again after fixing this.